feat(kibana-app): add troubleshooting for app version conflict (#3789)
wazuh/wazuh-documentation: source/_templates/common/clear_cache.rst

.. Copyright (C) 2021 Wazuh, Inc.

.. tabs::

  .. group-tab:: Chromium browser

>     #. Open the browser **DevTools**: press F12.

Reviewer: ```suggestion
    #. Open the web browser and press F12 to access the **DevTools” pane.

```

>     #. Go to the **network** tab: check the ``Disable cache`` option.

Reviewer: ```suggestion
    #. Click the **Network** tab and then check the ``Disable cache`` option.
```

      .. thumbnail:: ../../images/kibana-app/troubleshooting/disable_cache_chromium.png
        :title: Disable cache Chromium
        :align: left
        :width: 100%

  .. group-tab:: Firefox browser

    #. Open the browser **DevTools**: press F12.

>     #. Go to the **network** tab: check the ``Disable cache`` option.

Reviewer: ```suggestion
    #. Click the **Network** tab and then check the ``Disable cache`` option.
```
